src/routes/swagger/apis/tokenMint.ts

```diff
@@ -0,0 +1,86 @@
+/**
+ * @swagger
+ * /token/mint:
+ *   post:
+ *     summary: Mint token
```

Comment:
Mints a new token and transfers the tokens to the specified destination address.